h2/src/main/org/h2/expression/function/CSVWriteFunction.java

```diff
@@ -43,8 +44,10 @@ public Value getValue(SessionLocal session) {
             String escapeCharacter = getValue(session, 5);
             String nullString = getValue(session, 6);
             String lineSeparator = getValue(session, 7);
+            String quotedNulls = getValue(session, 8);
```

Comment:
I don't see any good reasons to quote nulls in `CSVWRITE`. I think only `CSVREAD` need this option.

Reply:
Removed. However, since `csv.setOptions` is used above for the non-legacy parsing, the option is still supported for writing (and I use it in my tests to ensure it is correctly written).